### ISSUE TITLE
Rider doesn't respect setting: fsharp_multi_line_lambda_closing_newline

### ISSUE THREAD

**Issue by panmau** [JetBrains/resharper-fsharp#297]
Rider version 2021.2 EAP 8 (and previous versions) doesn't respect the `fsharp_multi_line_lambda_closing_newline` formatting option from Fantomas which results in a mismatch between the formatting from Rider and the CLI tool.

My understanding is that the current EAP uses the newest version of this plugin from branch `net212` and this branch is on the latest Fantomas version (`4.5.0`) which I use locally.

# Repro Steps
1. Add [.editorconfig](https://github.com/panmau/rider-fantomas-repro/blob/main/.editorconfig) to project
```editorconfig
root = true

# Based on: https://github.com/G-Research/fsharp-formatting-conventions/blob/master/.editorconfig
# Difference is: disable_elmish_syntax=false and single_argument_web_mode=true
[*.fs]
fsharp_single_argument_web_mode=true
fsharp_space_before_uppercase_invocation=true
fsharp_space_before_member=true
fsharp_space_before_colon=true
fsharp_multiline_block_brackets_on_same_column=true
fsharp_newline_between_type_definition_and_members=true
fsharp_keep_if_then_in_same_line=true
fsharp_align_function_signature_to_indentation=true
fsharp_alternative_long_member_definitions=true
fsharp_disable_elmish_syntax=false
fsharp_multi_line_lambda_closing_newline=true
``` 
2. Add code that contains a lambda. For example:
```fsharp
let x =
    [ "Item"; "Item"; "Item" ]
    |> List.mapi (fun i x ->
        let index = i + 1
        sprintf "%s: %i" x index
    )
```
3. Format it with `Ctrl+Alt+L`

# Expected Result
```fsharp
let x =
    [ "Item"; "Item"; "Item" ]
    |> List.mapi (fun i x ->
        let index = i + 1
        sprintf "%s: %i" x index
    )
```

# Actual Result
```fsharp
let x =
    [ "Item"; "Item"; "Item" ]
    |> List.mapi
        (fun i x ->
            let index = i + 1
            sprintf "%s: %i" x index)
```

# Repository
I have made a small repo that contains minimal code for reproducing this error: https://github.com/panmau/rider-fantomas-repro

# Environment
Rider version: Rider 2021.2 EAP 8
Fantomas CLI: 4.5.0
OS: Manjaro Linux 21.1.0 Pahvo

**Comment by DedSec256** [comment]
Hi, thanks for the issue! We do not currently support this setting and are planning to add support before the release of 2021.2

**Comment by auduchinok** [comment]
@panmau Thanks for such a comprehensive report!

Some settings weren't added during recent Fantomas updates, and we currently have to do it manually during updates. It's too late to merge a fix to 2021.2 release, but https://github.com/JetBrains/resharper-fsharp/pull/298 is going to be included in a bug fix update. The logic implemented in https://github.com/JetBrains/resharper-fsharp/pull/261 allows us not to maintain the settings list altogether, so it'll be harder to lose a setting like this in 2021.3 and going forward.

**Comment by panmau** [comment]
@auduchinok Thank you for taking a look at this and the continued improvements regarding the integration of Fantomas in Rider.
I'm looking forward to the release of the bug fix!

**Comment by auduchinok** [comment]
@panmau That's kudos to @DedSec256 really, I've only shared a bit of the context in the comment. 🙂

**Comment by auduchinok** [comment]
Fixed in https://github.com/JetBrains/resharper-fsharp/pull/298.

**Comment by panmau** [comment]
Really great to see that the fix is released in EAP 9. Thanks for fixing this!

**Comment by panmau** [comment]
@auduchinok / @DedSec256 I tried it out in Rider.2021.2 and I have the behaviour that I originally described in the bug report.
Are you able to reproduce it? 
Could it be that the fix is not included in the new release?

**Comment by auduchinok** [comment]
@panmau It was merged to the main branch, and not to release one, so it's going to be included in the bugfix release. It seems for some reason the last EAP could've used that branch instead.

@DedSec256 Could you verify the nightly builds include it?

**Comment by DedSec256** [comment]
@panmau, @auduchinok, 
yes, indeed, the fix accidentally got into EAP 9. It didn't make it into the 2012.2 release, and there are currently no nightly builds, but the fix is planned for the 2021.2 bugfix release. Sorry for the misleading situation.

**Comment by panmau** [comment]
@DedSec256 Thanks for the quick clarification.

**Comment by DedSec256** [comment]
@panmau, it looks like the fix made it to the nightly 2021.2 build (:
